templates/jobs/jobs.html

```diff
@@ -10,6 +10,10 @@
                     <h2>
                         Job offers
                     </h2>
+                    <p>
+                        Working as a software developer is a great way to become one. There is lots of companies out there who are hiring, but how to tell which one is good? We talked to a lot of them and here you can find offers from companies who hire beginners and want to train them. They’re not targeting women only, but they do care about diversity in a workplace. We trust they do the right things.
```

Comment:
Working as a software developer is a great way to become one. There are lots of companies out there who are hiring, but how can you tell which ones are good? We have talked to a lot of them and here you can find offers from companies who hire beginners and want to train them. They’re not targeting women only, but they do care about diversity in a workplace. We trust they do the right things.